4-css-grid/style.css

```diff
@@ -0,0 +1,35 @@
+.grid {
+    display: grid;
+    grid-template-columns: 250px 250px 250px;
+    grid-template-rows: 250px 250px;
+    grid-gap: 20px;
+}
+
+div {
```

Comment:
you could also create a classname and apply it to the elements: `<div class="box A">A</div>` and by this way you don't force the markup to match your styles.